strr-examiner-web/app/pages/registration/[registrationId].vue

```diff
@@ -41,7 +41,7 @@ const handleRegistrationAction = (
 
   if (action === RegistrationActionsE.CANCEL) {
     actionFn = updateRegistrationStatus
-    additionalArgs = [RegistrationStatus.CANCELLED]
+    additionalArgs = [RegistrationStatus.CANCELLED, emailContent.value.content]
```

Comment:
Should this be also conditionally added? Or simply passing empty content will be enough?